src/System.Numerics.Vectors/src/CompatibilitySuppressions.xml

```diff
@@ -0,0 +1,60 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!-- https://learn.microsoft.com/en-us/dotnet/fundamentals/package-validation/diagnostic-ids -->
+<Suppressions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
+  <Suppression>
+    <DiagnosticId>CP0001</DiagnosticId>
+    <Target>T:System.Numerics.Matrix3x2</Target>
+    <Left>lib/netstandard2.0/System.Numerics.Vectors.dll</Left>
+    <Right>lib/net462/System.Numerics.Vectors.dll</Right>

```

Comment:
You need to add typeforwards for all these types.  Previously the partial facade infrastructure was doing that, but we're not using that anymore, so you'll need a netframework specific source-file that forwards each of these types.

Reply:
Is this something that I can generate with a command, or do I need to do this manually?

Reply:
GenAPI and GenFacades can both produce it.  Probably taking us longer to talk about it than just write the 7 lines...  Here's an intermediate file from dotnet/runtime "C:\src\dotnet\runtime\artifacts\obj\System.Numerics.Vectors\Debug\net9.0\System.Numerics.Vectors.Forwards.cs"
```c#
[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Numerics.Matrix3x2))]
[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Numerics.Matrix4x4))]
[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Numerics.Plane))]
[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Numerics.Quaternion))]
[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Numerics.Vector2))]
[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Numerics.Vector3))]
[assembly: System.Runtime.CompilerServices.TypeForwardedTo(typeof(System.Numerics.Vector4))]
```

Reply:
Added.